src/libraries/Microsoft.Extensions.Logging.Console/src/ContainerUtils.cs

```diff
@@ -0,0 +1,54 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Runtime.InteropServices;
+
+namespace Microsoft.Extensions.Logging.Console
+{
+    internal static class ContainerUtils
+    {
+        private static Lazy<bool> _isContainer = new Lazy<bool>(IsProcessRunningInContainer);
```

Comment:
>Lazy [](start = 53, length = 4)

I don't think we need to lazy initiaalization here. this class will get loaded when we call ContainerUtils.IsContainer. 

just make 


```C#
public static bool IsContainer { get; } = IsProcessRunningInContainer();
```